### PR TITLE
Code the /tos page (Fixes #10763)

### LINKED ISSUE #10763: Migrate /tos from pocket-marketing-pages
Moving the [/tos](https://www.getpocket.com/en/tos) page from pocket-marketing-pages to bedrock

### PR DESCRIPTION
**Pull request by nathan-barrett**
## Description
Coded the /tos page
## Issue / Bugzilla link
#10763
## Testing
http://localhost:8000/external/pocket/tos

### CODE REVIEW

File: bedrock/externalpages/templates/externalpages/pocket/tos.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    </section>
+
+    <section id="copyright">
+      <h2>Copyright infringement</h2>
+      <p>We will keep your information only for as long as we need it to provide you services, manage our business or as required by law or contract.</p>
```
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/tos.html:93]: This seems to be the wrong text seen here. The first paragraph is missing in this section, and I don't think that first text in the `p` tag is available on the original page?

File: bedrock/externalpages/templates/externalpages/pocket/tos.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
...of-date information, inaccuracies, typographical or other errors. notwithstanding any other provision of this agreement, we reserve the right to change, suspend, remove, or disable access to any of the pocket technologies, content, or other materials comprising a part of the pocket technologies at any time without notice. in no event will we be liable for making these changes. as a registered user in good standing, you may be provided with limited access to the videos, articles or other content saved to pocket. such access is provided as an accommodation only, and the items you save with pocket may be deleted at any time and for any reason. we do not warrant, and will not have any liability or responsibility for, your use of pocket or the products or services we provide. we may also impose limits on the use of or access to certain features or portions of the pocket technologies, for any reason and without notice or liability. our website, products and services may be unavailable from time to time due to mechanical, telecommunication, software, and other failures. we cannot predict or control when such downtime may occur and cannot control the duration of such downtime.</strong></p>
```
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/tos.html:112]: Wrong text is displayed here, you displayed the exact text as the paragraph above

File: media/css/externalpages/pocket/components/privacy-tos.scss [hunk 3]
```diff
@@ -44,6 +45,10 @@
 
     p {
         color: $color-text-secondary;
+
+        .uppercase {
+            text-transform: uppercase;
+        }
```
Comment (reemhamz) [inline media/css/externalpages/pocket/components/privacy-tos.scss:51]: Should have mentioned this in the `privacy` page PR, but let's also add a `&:focus, &:active` to the anchor tag, that way when people are tabbing through the page, they will also see the color change and whatnot

**Review by reemhamz — CHANGES_REQUESTED** [review]
Found a few mistakes in some `p` tags but it's to be expected when we're copy-pasting so much text! 🚀✨🪐

File: bedrock/externalpages/templates/externalpages/pocket/tos.html [hunk 4]
```diff
@@ -109,7 +109,7 @@ <h2>Proprietary rights notices</h2>
...
...et technologies. you acknowledge that we do not have control over your use of the pocket technologies, and we do not warrant the performance or results that may be obtained through your use of the pocket technologies. you assume all risks and responsibility for your use of the pocket technologies and for any loss of or errors in any data or information. to the full extent permissible by applicable law, we disclaim all warranties, express or implied, including, but not limited to, the implied warranties of merchantability, fitness for a particular purpose, and noninfringement, and any warranties arising through course of dealing or usage of trade. without limiting the foregoing, we neither warrant nor represent that your use of the pocket technologies will not infringe the rights of any third parties, nor that the pocket technologies will be available for your access or use, nor that operation of the pocket technologies will be error free or uninterrupted. please note that some jurisdictions may not allow the exclusion of implied warranties, so some of the above exclusions may not apply to you. in addition, you may also have other rights which vary from state to state."</strong></p>
```
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/tos.html:112]: Remove start and end quotes for this `p` tag

**Review by reemhamz — APPROVED** [review]
One last nit and it's good to merge. Good job copy pasting so much data and not going crazy! r+wc 🔑🧘🏻‍♀️

File: bedrock/externalpages/templates/externalpages/pocket/tos.html (hunk 4 above)
Comment (nathan-barrett) [reply bedrock/externalpages/templates/externalpages/pocket/tos.html:112]: oh thanks! you the best

**Pull request merged**